tests/zfs-tests/tests/functional/cli_root/zfs_upgrade/zfs_upgrade.kshlib

```diff
@@ -42,11 +42,22 @@ set -A spa_create_versions 9 15 24
 set -A zpl_upgrade_versions 3 4  5
 set -A spa_upgrade_versions 9 15 24
 
+function get_pool_version #pool
+{
+	ver=$(get_pool_prop version $1)
+
+	if [ "$ver" = "-" ]; then
+		ver="5000"
```

Comment:
I'm sorry for derailing this, i was wrongfully thinking of filesystem version, not pool version.

Reply:
@loli10K You didn't derail anything. Thank you for your assistance! :)